bunsen/bunsen-core-r4/src/main/java/com/cerner/bunsen/definitions/r4/R4StructureDefinitions.java

```diff
@@ -286,11 +270,7 @@ private <T> List<StructureField<T>> elementToFields(DefinitionVisitor<T> visitor
 
     String elementName = DefinitionVisitorsUtil.elementName(element.getPath());
 
-    if (shouldTerminateRecursive(visitor, rootDefinition, element, stack)) {
-
-      return Collections.emptyList();
-
-    } else if (element.getMax().equals("0")) {
+    if (element.getMax().equals("0")) {
```

Comment:
can we have a comment explaining what this if statement means?

Reply:
Hmm, there is a comment on the next line which explains it, isn't it clear?
BTW, I think this is for an edge case which should not happen for well-formed `ElementDefinition`s, i.e., defining an element but setting its `max` to zero.

Reply:
oh, I didnt realise.